lib/runtime/runtime.go

```diff
@@ -168,15 +165,3 @@ func (r *Runtime) malloc(size uint32) (uint32, error) {
 func (r *Runtime) free(ptr uint32) error {
 	return r.allocator.Deallocate(ptr)
 }
-
-func decodeToInterface(in []byte, t interface{}) (interface{}, error) {
```

Comment:
was this unused?

Reply:
It's no longer used since we're using the Decode interface for that type.